shared/router-v2/tab-bar/container.desktop.js

```diff
@@ -36,6 +37,9 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
     if (ownProps.selectedTab === Tabs.peopleTab && tab !== Tabs.peopleTab) {
       dispatch(PeopleGen.createMarkViewed())
     }
+    if (ownProps.selectedTab === Tabs.chatTab && tab !== Tabs.chatTab) {
```

Comment:
This is checking for when we switch away from the chat tab, intended?

Reply:
nope fixed